Deploy/Functions/New-ModuleChangeLog.ps1

```diff
@@ -7,9 +7,13 @@ Function New-ModuleChangelog {
         , [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, Position = 3)][ValidateNotNullOrEmpty()][System.String]$Improvements
         , [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, Position = 4)][ValidateNotNullOrEmpty()][System.String]$BugFixes
     )
+    $todayDate = Get-Date -UFormat "%B %d, %Y"
```

Comment:
Just adding a change here, this will generate today's date the same way we test for it in our `build.Tests.ps1`. This was annoying me, addressing this tech debt here.